### PR TITLE
[UXE-2053] feat: Insert properties in the event create edge application

### PR DESCRIPTION
**Pull request by pauloSF0**
## Pull Request
[UXE-2053]

### What is the new behavior introduced by this PR?
Insert properties in the event create edge application

### Does this PR introduce breaking changes?
- [x] No
- [ ] Yes 

### Does this PR introduce UI changes? Add a video or screenshots here.
Captura de Tela 2024-03-08 às 15 24 06

### Does it have a link on Figma?

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (fix, feat, test, etc)
- [ ] User inputs are sanitized/protected against malicious attacks
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari

[UXE-2053]: https://aziontech.atlassian.net/browse/UXE-2053?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by lucasmendes21 — APPROVED** [review]

File: src/templates/create-modal-block/index.vue
Comment (william-tome) [inline src/templates/create-modal-block/index.vue:1]: Revisitar esse nome criado. create/create fica bem estranho

File: src/router/routes/edge-application-routes/index.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ export const edgeApplicationRoutes = {
       }
     },
     {
-      path: 'create',
+      path: 'create/:origin?',
```
Comment (william-tome) [inline src/router/routes/edge-application-routes/index.js:34]: Me parece melhor seguir pelo caminho de usar queryParams, pois assim seria mais descritivo evitando gerar dúvidas em quem lê.

File: src/templates/create-modal-block/index.vue [hunk 2]
```diff
@@ -47,7 +47,7 @@
     },
     {
       label: 'Edge Application',
-      to: '/edge-applications/create',
+      to: '/edge-applications/create/create',
```
Comment (william-tome) [inline src/templates/create-modal-block/index.vue:50]:
```suggestion
      to: '/edge-applications/create?origin=create',
```

File: src/views/RealTimeEventsTieredCache/ListView.vue [hunk 3]
```diff
@@ -148,6 +148,7 @@
     title="No logs have been found for this period."
     description="Use the filter to change time range and variables, or create a new edge application with Tiered Cache configurations. Logs are displayed once there are incoming requests and traffic."
     createButtonLabel="Edge Application"
+    createPagePath="/edge-applications/create/realTimeEvents"
```
Comment (william-tome) [inline src/views/RealTimeEventsTieredCache/ListView.vue:151]:
```suggestion
    createPagePath="/edge-applications/create?origin=realTimeEvents"
```

File: src/views/EdgeApplications/ListView.vue [hunk 4]
```diff
@@ -98,7 +98,7 @@
         title="No edge applications have been created"
         description="Click the button below to initiate the setup process and create your first edge application."
         createButtonLabel="Edge Application"
-        createPagePath="/edge-applications/create"
+        createPagePath="/edge-applications/create/list"
```
Comment (william-tome) [inline src/views/EdgeApplications/ListView.vue:101]:
```suggestion
        createPagePath="/edge-applications/create?origin=list"
```

File: src/views/EdgeApplications/ListView.vue [hunk 5]
```diff
@@ -83,7 +83,7 @@
         v-if="hasContentToList"
         pageTitleDelete="Edge Application"
         addButtonLabel="Edge Application"
-        createPagePath="/edge-applications/create"
+        createPagePath="/edge-applications/create/list"
```
Comment (william-tome) [inline src/views/EdgeApplications/ListView.vue:86]:
```suggestion
        createPagePath="/edge-applications/create?origin=list"
```

File: src/views/EdgeApplications/CreateView.vue [hunk 6]
```diff
@@ -84,7 +86,9 @@
 
   const handleTrackCreation = () => {
     tracker.productCreated({
-      productName: 'Edge Application'
+      productName: 'Edge Application',
+      from: route.params.origin,
```
Comment (william-tome) [inline src/views/EdgeApplications/CreateView.vue:90]:
```suggestion
      from: route.query.origin,
```

**Review by william-tome — CHANGES_REQUESTED** [review]

**Review by william-tome — DISMISSED** [review]

**Review by william-tome — APPROVED** [review]

**Review by william-tome — APPROVED** [review]

**Review by lucasmendes21 — APPROVED** [review]

**Review by peterpaulo-azion — APPROVED** [review]

**Pull request merged**